xcode/Subconscious/Shared/Services/Noosphere/NoosphereService.swift

```diff
@@ -140,14 +140,13 @@ final class NoosphereService: SphereProtocol, SphereIdentityProtocol {
         guard let identity = self._sphereIdentity else {
             throw NoosphereServiceError.defaultSphereNotFound
         }
+        
         let noosphere = try noosphere()
         logger.debug("init Sphere with identity: \(identity)")
-        let sphere = try Sphere(
+        return try Sphere(
             noosphere: noosphere,
             identity: identity
         )
-        self._sphere = sphere
```

Comment:
I think this removal was unintended? We need to cache the user's sphere to `self._sphere` here.

Reply:
Good catch, restored.